test/core/ExamplesUnitTests/TribitsExampleProject2_Tests.cmake

```diff
@@ -541,10 +541,24 @@ TribitsExampleProject2_explicit_tpl_vars(SHARED)
 ########################################################################
 
 
-function(TribitsExampleProject2_find_package  sharedOrStatic)
+function(TribitsExampleProject2_find_package  sharedOrStatic  package1TribitsOrRawCMake)
 
   TribitsExampleProject2_test_setup_header()
 
+  if ( (package1TribitsOrRawCMake STREQUAL "") OR
+      (package1TribitsOrRawCMake STREQUAL "PACKAGE1_USE_TRIBITS")
+    )
+    set(package1UseRawCMakeArgs "")
+    set(testNameSuffix "")
+    set(package1ConfiRegex "Configuring package Package1 as full TriBITS package")
```

Comment:
Was this supposed to be `package1ConfigRegex`?

Reply:
Resolved in new commit bad293d70d022628d8d23ba70822b8696340e41c.